src/index/IndexImpl.Text.cpp

```diff
@@ -1036,6 +1036,20 @@ Index::WordEntityPostings IndexImpl::getEntityPostingsForTerm(
   return resultWep;
 }
 
+// _____________________________________________________________________________
+Index::WordEntityPostings IndexImpl::getUnadjustedEntityPostingsForTerm(
+    const string& term) const {
+  LOG(DEBUG) << "Getting unadjusted entity postings for term: " << term << '\n';
+  Index::WordEntityPostings resultWep;
+  auto optTbmd = getTextBlockMetadataForWordOrPrefix(term);
+  if (!optTbmd.has_value()) {
+    return resultWep;
+  }
+  const auto& tbmd = optTbmd.value().tbmd_;
+  resultWep = readWordEntityCl(tbmd);
```

Comment:
`return readWordEntityCl(tbmd)`

Reply:
Then you can get rid of the above function.

Also: 
(Maybe for later) add a `TODO` that this function should return an `IdTable` directly.